locations/spiders/quickchek_us.py

```diff
@@ -0,0 +1,31 @@
+from scrapy import Request, Spider
+
+from locations.dict_parser import DictParser
+from locations.hours import DAYS, OpeningHours
+
+
+class QuickChekUSSpider(Spider):
+    name = "quickchek_us"
+    item_attributes = {"brand": "QuickChek", "brand_wikidata": "Q7271689"}
+    allowed_domains = ["quickchek.com"]
+    start_urls = ["https://quickchek.com/wp-admin/admin-ajax.php"]
+
+    def start_requests(self):
+        for url in self.start_urls:
+            headers = {
+                "Content-Type": "application/x-www-form-urlencoded; charset=UTF-8",
+                "X-Requested-With": "XMLHttpRequest",
+            }
+            yield Request(
+                url=url, method="POST", body="action=get_sorted_locations&dist=100000&lat=0&lng=0", headers=headers
+            )
```

Comment:
```suggestion
            yield FormRequest(
                url=url,
                formdata={"action": "get_sorted_locations", "dist": "100000", "lat": "0", "lng": "0"},
                headers={"X-Requested-With": "XMLHttpRequest"},
            )
```
I'm 99% sure FormRequest will handle the headers and body for us. But I've been wrong before.

Reply:
OK that works, unlike https://github.com/alltheplaces/alltheplaces/pull/5911 where the API couldn't handle `"0"` (string) being sent as a parameter value.

Reply:
Thanks!